terasoluna-gfw-functionaltest-web/pom.xml

```diff
@@ -154,6 +154,13 @@
       <scope>test</scope>
     </dependency>
     <!-- == End Unit Test == -->
+
+    <!-- == Begin logback-db == -->
+    <dependency>
+      <groupId>ch.qos.logback.db</groupId>
+      <artifactId>logback-classic-db</artifactId>
+    </dependency>
+    <!-- == End logback-db == -->
```

Comment:
Why do you need to re-define the domain dependencies?

Reply:
Oh...No redefinition necessary.

Reply:
Fixed 16ad9f9d0a10b10f96c8e6a31e426eb18d66af96 in #986.